controllers/datadogagent/poddisruptionbudget.go

```diff
@@ -33,37 +35,74 @@ type (
 	pdbBuilder func(dda *datadoghqv1alpha1.DatadogAgent) *policyv1.PodDisruptionBudget
 )
 
+type (
+	pdbV1Beta1Builder func(dda *datadoghqv1alpha1.DatadogAgent) *policyv1beta1.PodDisruptionBudget
```

Comment:
Both should be in a single `type ()` or not use `()`

Reply:
Fixed.